tablet_integration_test.go

```diff
@@ -69,7 +69,9 @@ func TestTablets(t *testing.T) {
 			}
 		}
 
-		assertEqual(t, "queriedHosts", 1, queriedHosts)
+		if queriedHosts != 1 {
+			t.Fatalf("trace should show host only once but it showed it %d, hosts: %s trace:\n%s", queriedHosts, hostAddresses, buf.String())
```

Comment:
I meant: "trace should show only one host, but it showed %d, hosts: %s trace: \n%s"
Host can be present in the trace multiple times but it has to be the same host

Reply:
Thanks, changed language again, take a look.

Reply:
It would be nice to have similar change in the second test in this file, could you add it?